completions/clang

```diff
@@ -0,0 +1,81 @@
+_clang_filedir()
+{
+    # _filedir function provided by recent versions of bash-completion package is
+    # better than "compgen -f" because the former honors spaces in pathnames while
+    # the latter doesn't. So we use compgen only when _filedir is not provided.
+    _filedir 2> /dev/null || COMPREPLY=( $( compgen -f ) )
+}
+
+_clang()
+{
+    local cur prev words cword arg flags w1 w2
+    # If latest bash-completion is not supported just initialize COMPREPLY and
+    # initialize variables by setting manualy.
+    _init_completion -n 2> /dev/null
```

Comment:
We know latest bash-completion is installed if this file is, please simplify accordingly.